README.md

```diff
@@ -319,8 +321,9 @@
 
 ## Peephole Optimization
 
-- [`egg`](https://crates.io/crates/egg) Egg is a flexible, high-performance e-graph library 
+- [`egg`](https://crates.io/crates/egg) Egg is a flexible, high-performance e-graph library
 - [`peepmatic`](https://crates.io/crates/peepmatic) A DSL for peephole optimizations and compiler for generating peephole optimizers from them
+- [`egglog`](https://crates.io/crates/egglog) Egglog is a language that combines the benefits of equality saturation and datalog. It can be used for analysis, optimization, and synthesis of programs. It is the successor to the popular Rust Library egg.
```

Comment:
I wasn't able to find anywhere that egg or egglog use this successor verbiage, so it seems a bit strong to me.
In the [Egg README](https://github.com/egraphs-good/egg/blob/main/README.md) they describe it as

> an alternative approach to equality saturation based on Datalog.

So I would suggest something along the lines of 
"It provides an alternative approach to the popular Rust Library egg."

Reply:
I read it here:
<img width="940" alt="image" src="https://github.com/Kixiron/rust-langdev/assets/15143039/f46663c6-6417-4975-8e94-4e95a631794f">
I can remove it though

Reply:
Aha, was hiding in the Cargo.toml not the README.